### PR TITLE
python310Packages.sphinxcontrib-details-directive: init at 0.1.0

### PR DESCRIPTION
**Pull request by toastal**
###### Description of changes
https://github.com/sphinx-contrib/sphinxcontrib-details-directive/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eL1qZ/eGohhoq/DmpZczQNfnpv1xsYkN6chW+Sh3s4s=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+  ];
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix:23]: Why is this disabled?

Comment (toastal) [reply pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix]: The `sphinxcontrib-*` file I copied from as a template had it. Removed.

File: pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eL1qZ/eGohhoq/DmpZczQNfnpv1xsYkN6chW+Sh3s4s=";
+  };
+
+  propagatedBuildInputs = [
+    sphinx
+  ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix:20]:
```suggestion

  nativeCheckInputs = [
    pytestCheckHook
  ];

```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix (hunk 2 above)
Comment (toastal) [reply pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix:20]:
```
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.2.1, pluggy-1.0.0
rootdir: /build/sphinxcontrib-details-directive-0.1.0, configfile: setup.cfg
collected 0 items                                                              

============================ no tests ran in 0.01s =============================
/nix/store/blpvf60m29q02c0lc5fyhim30ma4y1vv-stdenv-linux/setup: line 1596: pop_var_context: head of shell_variables not a function context
```
I'm not sure what to do with this and it fails to run.

Comment (SuperSandro2000) [reply pkgs/development/python-modules/sphinxcontrib-details-directive/default.nix:20]: I think the tests are not in the sdist and we would need to fetch from github to run them.

**Comment by toastal** [comment]
I’m no longer trying to use this.

**Pull request closed without merging**